// Copyright 2021 Google LLC
//
// Licensed under the Apache License, Version 2.0 (the "License");
// you may not use this file except in compliance with the License.
// You may obtain a copy of the License at
//
//      http://www.apache.org/licenses/LICENSE-2.0
//
// Unless required by applicable law or agreed to in writing, software
// distributed under the License is distributed on an "AS IS" BASIS,
// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
// See the License for the specific language governing permissions and
// limitations under the License.

syntax = "proto3";

package fcp.tracing.test;

message ApiMessageWithoutContext {
  string contents = 1;
}

message ApiMessageWithContext {
  string contents = 1;
  string tracing_context = 2;
}

message ApiMessageWithContextBytes {
  string contents = 1;
  bytes tracing_context = 2;
}

message ApiMessageWithContextInt {
  string contents = 1;
  int32 tracing_context = 2;
}

message TestTracingContext {
  int32 first = 1;
  int32 second = 2;
}
